apps/condo/domains/common/components/MenuItem.tsx

```diff
@@ -170,5 +170,5 @@ export const MenuItem: React.FC<IMenuItemProps> = (props) => {
 
     const nextjsLink = !path || disabled ? menuItem : makeLink(menuItem, path)
 
-    return toolTipDecorator ? toolTipDecorator({ element: nextjsLink, placement: 'right' }) : nextjsLink
+    return <div>{toolTipDecorator ? toolTipDecorator({ element: nextjsLink, placement: 'right' }) : nextjsLink}</div>
```

Comment:
Guess it's already fixed, you should rebase and remove this wrapper:
https://github.com/open-condo-software/condo/blob/master/apps/condo/domains/common/components/MenuItem.tsx#L98

Reply:
I already have it. It's a different issue. I'll add a comment

Reply:
yep, you are right! you already fix it
I'll drop it